lib/ApiClient/ApiClient.php

```diff
@@ -370,6 +370,8 @@ public function syncAds(array $adIds)
             throw new \Exception('A hirdetés ID-k lekérése nem sikerült', 0, $e);
         }
 
+        $adIds = $this->normalizeAdIds($adIds);
```

Comment:
@ggergo hol okoz hibát? látom lentebb össze van diffelve az `$ownIds` és az `$adIds`, de az array_diff egyébként is string-re castolva típusosan hasonlít:

https://sandbox.onlinephpfunctions.com/code/2bebef410220427f27868c9193b13a5cd92b296b

[Two elements are considered equal if and only if (string) $elem1 === (string) $elem2. That is, when the string representation is the same.](https://www.php.net/manual/en/function.array-diff.php)

Reply:
ezt jól benéztem :) legközelebb nem hiszek a rövid storm suggestion-nek